dashboard/src/t5gweb/utils.py

```diff
@@ -234,6 +238,7 @@ def set_defaults():
     defaults["slack_channel"] = ""
     defaults["max_jira_results"] = 1000
     defaults["max_portal_results"] = 5000
+    defaults["sla_settings"] = ""
```

Comment:
Everything looks okay to me, but if the variable is set to None (say if someone didn't update the configmap or something) then we'll hit an error later on when:
~~~
 {% if new_comments[account][status][card]['account'] in sla_settings["partners"] %}
~~~
due to `TypeError: 'NoneType' object is not subscriptable`

maybe set the default to something sane like: `sla_config={"days": {"Urgent": 14,"High": 20,"Normal": 90,"Low": 180},"partners": []}`? also provides a better example of what it should look like if someone is reading to code. then use the default instead of `None` when checking the environment variable.

Reply:
ack, changed, thanks!